harness/determined/common/experimental/determined.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         cert_path: Optional[str] = None,
         cert_name: Optional[str] = None,
         noverify: bool = False,
+        try_reauth: bool = True,
```

Comment:
Ugh, is there any way to solve this problem without exposing `try_reauth` to the end user?

Or, I suppose, is there any reason why an end user would use the python SDK and need to know that `try_reauth` exists?  This feels like like a leaky abstraction

Reply:
could be called `--non-interactive`?